training/distributed_training/pytorch/model_parallel/GPT2_huggingface_trainer/submit_smp_trainer.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<p>I'm not sure if the <code>checkpoint_dir</code> is set correctly here. We have it documented here: https://docs.aws.amazon.com/sagemaker/latest/dg/model-parallel-extended-features-pytorch-checkpoint.html</p><p>Please double check it with Fei.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3577/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Yeah, the `checkpoint_dir` is set to the directory that SageMaker monitors for checkpoints. Hugging Face Trainer should automatically be saving/loading from the proper folders with smp!